js/widgets.js

```diff
@@ -0,0 +1,75 @@
+jQuery( document ).ready(function( $ ) {
+	var widgets_container, widgets_selector;
+
+	/**
+	 * Prepend widget titles with a flag once a language is selected.
+	 * @param {object} widget The widget element.
+	 * @return {void} Nothing.
+	 */
+	function add_flag( widget ) {
+		widget = $( widget );
+		var title = $( '.widget-top .widget-title h3', widget ),
+			icon = $( '.pll-lang-choice option:selected', widget ).attr( 'flag' );
+
+		if ( icon ) {
+			var current = $( '.pll-lang img', title );
+			if ( current.length ) {
+				current.attr( 'src', icon );
+			} else {
+				var img  = '<img src="' + icon + '">',
+					flag = '<span class="pll-lang">' + img + ' &nbsp; </span>';
+				title.prepend( flag );
+			}
+		} else {
+			$( '.pll-lang', title ).remove();
+		}
+	}
+
+	if ( 'undefined' !== typeof wp.customize ) {
+
+		widgets_container = $( '#customize-controls' );
+		widgets_selector  = '.customize-control .widget';
+
+		/**
+		 * WP Customizer add control listener.
+		 *
+		 * @link https://wordpress.stackexchange.com/questions/256536/callback-after-wordpress-customizer-complete-loading
+		 *
+		 * @param {object} control The control type.
+		 * @return {void} Nothing.
+		 */
+		function customize_add_flag( control ) {
+			if ( ! control.extended( wp.customize.Widgets.WidgetControl ) ) {
+				return;
+			}
+
+			/*
+			 * Make sure the widget's contents are embedded; normally this is done
+			 * when the control is expanded, for DOM performance reasons.
+			 */
+			control.embedWidgetContent();
+
+			// Now we know for sure the widget is fully embedded.
+			add_flag( control.container.find( '.widget' ) );
+		}
+		wp.customize.control.each( customize_add_flag );
+		wp.customize.control.bind( 'add', customize_add_flag );
```

Comment:
Could be better to change this to `.on()`?
http://api.jquery.com/bind/

Reply:
It could be also better to change `jQuery( document ).ready(function( $ ) {` to just `jQuery( function( $ ) {`
https://jquery.com/upgrade-guide/3.0/#deprecated-document-ready-handlers-other-than-jquery-function

Reply:
@chesio Good one, new handler is set:
```js
( function( $ ) {
    // Code
} ( jQuery ) );
```

The use of `.on()` is not possible since `wp.customize.control` is not a jQuery object.